docker/Dockerfile-envoy

```diff
@@ -12,7 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM python:3.6-buster
+FROM frolvlad/alpine-glibc:alpine-3.14_glibc-2.33
```

Comment:
Which team owns and maintains this image?

Reply:
+1, are you ok introducing a dependency on a community maintained image?

FYI license for image is OK, it is MIT: https://github.com/Docker-Hub-frolvlad/docker-alpine-glibc/blob/master/LICENSE

But please check go/cloudimage. I am not sure if this falls under the pre-approved list. Might be worth reaching out to OSPO.

Another approach is to consider forking https://github.com/Docker-Hub-frolvlad/docker-alpine-glibc/blob/master/Dockerfile and building the image ourself, with the proper attribution to the source license and author. Please check with OSPO if this approach is allowed.

Reply:
I decided to just use the one build by frolvlad/alpine-glibc:  it has a community maintaining this, to upgrade and fix some bugs. it is better than we fork it

I added its LICENSE file in. It is MIT license.

Reply:
Depending on community image is OK as long as we have a maintainability story moving forward. I guess we'll just update this image from time to time after the community lands a new version?